src/main/java/de/doubleslash/keeptime/viewpopup/ViewControllerPopup.java

```diff
@@ -115,6 +107,15 @@ private void initialize() {
 
       Interval.registerCallBack(() -> projectsListViewController.tick());
```

Comment:
if we look at it like this, this seems dangerous, registering callbacks on an object, which is instantiated one line after the registration :)

Reply:
yes it is merged correctly